From 6c95ae2bf1c80530442a516f23b7cd8e82dcae12 Mon Sep 17 00:00:00 2001
From: Matej Habrnal <mhabrnal@redhat.com>
Date: Thu, 22 Jan 2015 02:23:21 +0100
Subject: [PATCH 88/91] event: don't run the 'reporter-bugzilla -h' on RHEL and
 CentOS

Running the 'reporter-bugzilla -h' makes sense only on Fedora because of bodhi.

Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>
---
 src/plugins/ccpp_event.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/ccpp_event.conf b/src/plugins/ccpp_event.conf
index 62ff08a..cd75ee2 100644
--- a/src/plugins/ccpp_event.conf
+++ b/src/plugins/ccpp_event.conf
@@ -71,7 +71,7 @@ EVENT=analyze_LocalGDB analyzer=CCpp
         # Run GDB plugin to see if crash looks exploitable
         abrt-action-analyze-vulnerability
         # Run GDB to genereate backtrace
-        abrt-action-analyze-ccpp-local --without-bodhi
+        abrt-action-analyze-ccpp-local --without-bz
 
 
 # Bugzilla requires nonempty duphash
-- 
1.8.3.1

